rest/config.go

```diff
@@ -69,6 +72,7 @@ const (
 
 // JSON object that defines the server configuration.
 type ServerConfig struct {
+	TLSVersion                 *TLSVersion              `json:"tlsversion,omitempty"`              // Set TLS Version
```

Comment:
This should be named 'tls_minimum_version' for clarity.  Updating the internal variable name might also be nice.